src/snapred/backend/data/DataFactoryService.py

```diff
@@ -4,27 +4,38 @@
 from snapred.backend.dao.InstrumentConfig import InstrumentConfig
 from snapred.backend.dao.ReductionState import ReductionState
 from snapred.backend.dao.RunConfig import RunConfig
+from snapred.backend.dao.state import PixelGroupingParameters
 from snapred.backend.dao.StateConfig import StateConfig
 from snapred.backend.data.LocalDataService import LocalDataService
+from snapred.backend.data.LocalWorkspaceDataService import LocalWorkspaceDataService
 from snapred.meta.decorators.Singleton import Singleton
 
 
 @Singleton
 class DataFactoryService:
     lookupService: "LocalDataService"  # Optional[LocalDataService]
+    workspaceService: "LocalWorkspaceDataService"  # Optional[LocalWorkspaceDataService]
```

Comment:
I'm unsure what this is

Reply:
it refers to [this](https://github.com/neutrons/SNAPRed/pull/162/files#diff-73b39fce4f81d3e680dff63f329246a6517b3f52c9f23348bc35769747d26d74) which is a splinter off of the LocalDataService to help manage its filesize.